src/libs/ajax.js

```diff
@@ -678,12 +678,12 @@ const Workspaces = signal => ({
       },
 
       listSnapshots: async (limit, offset) => {
-        const res = await fetchRawls(`${root}/snapshots?offset=${offset}&limit=${limit}`, _.merge(authOpts(), { signal }))
+        const res = await fetchRawls(`${root}/snapshots/v2?offset=${offset}&limit=${limit}`, _.merge(authOpts(), { signal }))
```

Comment:
This could be where the type mismatch is introduced. Rawls may be returning a snapshot ID in a field with the label "snapshot." If that were true, I might file a ticket against Rawls and make the correction here to contain the impact.